composer.json

```diff
@@ -64,7 +65,6 @@
         "symplify/phpstan-extensions": "^9.4.4",
         "symplify/phpstan-rules": "^9.4.4",
         "symplify/rule-doc-generator": "^9.4.4",
-        "symplify/vendor-patches": "^9.4",
```

Comment:
This fixes patch applying: 
![image](https://user-images.githubusercontent.com/924196/124460828-258da100-dd90-11eb-8b9c-59198a39fd54.png)


With `require-dev`, the patches were removed in here:

https://github.com/rectorphp/rector-src/blob/0a616c462d741fb581cf24f25a1bc1ff60668178/.github/workflows/build_scoped_rector.yaml#L35-L36

Reply:
Patching works and is deployed to `rector/rector` ↓ :tada: 

https://github.com/rectorphp/rector/commit/95fe045d3026b96bf4fbd01fe70b7951cf27e344#diff-d3533fe4f13c7c9566b3b99fd27ec2b263cd1b9c687ecb5fc2269585ad50b805